pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -176,6 +188,15 @@ def write_pep0(
             )
             self.emit_newline()
 
+        # PEPs by number
+        if is_pep0:
+            self.emit_title("Numerical Index")
+            self.emit_text(
+                "The :doc:`numerical index </numerical>` contains "
+                "a table of all PEPs :ref:`sorted by number <numerical-index>`."
```

Comment:
On second thoughts, these links are essentially duplicates, let's keep just one of them. Any preference which one?

Reply:
The first link. Though it is a toss up. I would assume a numerical index is sorted by default.

Reply:
```suggestion
                "The :doc:`numerical index </numerical>` contains "
                "a table of all PEPs, ordered by number."
```

or 

```suggestion
                "The :doc:`numerical index </numerical>` contains "
                "a table of all PEPs."
```